### DISCUSSION TITLE
Add LogScope by default as context when logging a custom event

### DISCUSSION THREAD

**Discussion by fgheysels** [arcus-azure/arcus.observability#186] [Q&A]
I have the following code:

```csharp
using (var logscope = logger.BeginScope(new Dictionary<string, object>()
{
    ["ShovelName"] = shovel.name
}))
{
     logger.LogWarning("Shovel {shovelname} is not running", shovel.name);
     logger.LogEvent("Shovel restarted");
}
```

When logging to the console, this gives the following output:

```
2021-03-05 16:15:29 [Warning] Shovel "vessel2cloud" is not running
2021-03-05 16:15:29 [Warning] Events "Shovel restarted" (Context: {})
```

I'd expect that the 'scope context' is automatically used as context when logging the event.   I don't know if it's a good idea to do so - and what should we do when you have 'scope' and provide a context argument to the LogEvent method; should these be merged then ?  

Next to that, I expected to see the scope - context in the output of the `LogWarning` as well.

**Comment by tomkerkhove** [comment]
The context you are seeing is just what is provided as a dictionary and is specific to our extensions.

If you [change the template of the console logger](https://github.com/serilog/serilog/wiki/Formatting-Output), you should be able to add the shovel name so that it starts showing it.

For example: `{Timestamp:yyyy-MM-dd HH:mm:ss}|{Level} => Shovel Name:{ShovelName} => {Message}{NewLine}{Exception}`
